setup/helpers/abs_helper.rb

```diff
@@ -754,26 +754,31 @@ def parse_abs_resource_hosts(abs_resource_hosts)
   #
   # @raise [RuntimeError] If provisioning is not successful
   #
-  # @return [String] The hostname of the provisioned host
+  # @return [Hash] The provisioned host
   #
   # @example
-  #   hostname = provision_host_for_role(role)
+  #   host = provision_host_for_role(role)
   #     or
-  #   hostname = provision_host_for_role(role, "c5.xlarge", "40")
+  #   host = provision_host_for_role(role, "c5.xlarge", "40")
   #
-  # TODO: spec test(s)
   def provision_host_for_role(role, size = AWS_SIZE, volume_size = AWS_VOLUME_SIZE)
     host_to_provision = get_host_to_provision(role, size, volume_size)
     abs_resource_hosts = get_abs_resource_hosts(host_to_provision)
     raise "Unable to provision host via ABS" unless abs_resource_hosts
 
+    # parse the JSON
     hosts = parse_abs_resource_hosts(abs_resource_hosts)
-    hostname = hosts[0]["hostname"]
 
-    puts "Successfully provisioned host - role: #{role}, hostname: #{hostname}"
+    # convert keys from strings to symbols (provision_pe_xl_nodes.rb expects symbols)
```

Comment:
Is this a bug in provision_pe_xl_nodes.rb?
It seems like we should standardize on what an "abs_host" hash looks like.  So, either provision_pe_xl_nodes.rb should be updated to use string keys _or_ `parse_abs_resource_hosts` should be updated to use symbol keys.

Reply:
Ah; I meant to create a ticket to standardize on using symbols as hash keys throughout the module. [SLV-676](https://tickets.puppetlabs.com/browse/SLV-676) has been created.